src/sentry/tasks/store.py

```diff
@@ -316,6 +317,16 @@ def save_event(cache_key=None, data=None, start_time=None, event_id=None, **kwar
                 key=project_key,
                 timestamp=start_time,
             )
+    else:
+        try:
+            project = Project.objects.get_from_cache(id=project_id)
+        except Project.DoesNotExist:
+            pass
+        else:
+            event_saved.send_robust(
+                project=project,
```

Comment:
Can we not pass along a project_id instead? What downstream from this actually needs the Project?

Reply:
the only thing we need is org id.. so i could query just for that and send that along too

Reply:
possibly worth mentioning, our other signals use the project model, and this is where i'll be using these: https://github.com/getsentry/getsentry/blob/master/getsentry/receivers/usage.py#L24